src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -196,11 +196,62 @@ private async Task ProcessParticipantDataAsync(IEnumerable<Participant> particip
         catch (Exception ex)
         {
             _logger.LogError(ex, "Error in ProcessParticipantDataAsync: {Message}", ex.Message);
-            await ProcessParticipantDataAsync(participants, participantUrl);
+            await ProcessParticipantDataAsync(subjects, participantUrl);
         }
     }
+        private ParticipantDto MapParticipantToParticipantDto(BssSubject subject)
+        {
+            return new ParticipantDto
+            {
+                NhsNumber = subject.nhs_number,
+                ScreeningName = "Breast Screening",
+                NextTestDueDate = subject.next_test_due_date,
+                NextTestDueDateCalculationMethod = subject.ntdd_calculation_method,
+                ParticipantScreeningStatus = subject.subject_status_code,
+                ScreeningCeasedReason = subject.reason_for_ceasing_code,
+                IsHigherRisk = GetIsHigherRisk(subject.is_higher_risk),
+                IsHigherRiskActive = GetIsHigherRiskActive(subject.is_higher_risk_active),
+                HigherRiskNextTestDueDate = subject.higher_risk_next_test_due_date,
+                HigherRiskReferralReasonCode = subject.higher_risk_referral_reason_code,
+                DateIrradiated = subject.date_irradiated,
+                GeneCode = subject.gene_code
+            };
+        }
+
+        private static short? GetIsHigherRisk(string isHigherRisk)
+        {
+            if (isHigherRisk.ToUpper() == "TRUE")
+            {
+                return (short)1;
+            }
+            else if (isHigherRisk.ToUpper() == "FALSE")
+            {
+                return (short)0;
+            }
+            else
+            {
+                return null;
+            }
+        }
+
+        private static short? GetIsHigherRiskActive(string isHigherRiskActive)
```

Comment:
These two methods work the same way as GetAppointmentMadeFlag, we should be able to use one method for all of them, it can be named something like ParseBooleanStringToShort

Reply:
Ready for review